plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1794,6 +1788,68 @@ public Optional<ConnectorOutputMetadata> finishInsert(
         return Optional.empty();
     }
 
+    private long getInsertCommitVersion(ConnectorSession session, DeltaLakeInsertTableHandle handle, List<DataFileInfo> dataFileInfos)
+            throws IOException
+    {
+        long createdTime = Instant.now().toEpochMilli();
```

Comment:
Should we move it closer to write logic. Is it important to have time order in sync with version? Currently we can have `x + 1` version creation time before `x` version.